azure-cli/synapse/Publish/Parameterize.ps1

```diff
@@ -0,0 +1,35 @@
+param (
+  [Parameter(Mandatory = $True)]
+  [ValidateNotNullOrEmpty()]
+  [string]
+  $environmentName = "dev",
```

Comment:
[Use pascal case for PowerShell parameters](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.2#use-pascal-case-for-parameter-names)